website/docs/configuring_clients.md

```diff
@@ -4,8 +4,186 @@ title: Configuring Clients
 ---
 
 ## Android
- - AEROGEAR-10136	
+ Before you begin configuring an Android application to receive push messages, you should have the following installed, available, or configured as appropriate.
+
+  * Android Studio
+  * Google Firebase Account
+  * Android Emulator or device with Google Play Services
+  * Unified Push with an Android Variant Configured
  
+### Overview
+Unified Push makes use of Google's Firebase Cloud Messaging service (FCM).  Before you can use push messages with Android you will need to create a Firebase project.  Android Studio can guide you through this process.  Once you have your Firebase project set up, you can enable Firebase in your Android application.  Like before, Android Studio has tools to facilitate this.  These steps are covered in the section *Firebase Configuration with Android Studio*.
+
+Once you have Firebase configured in your application, you can configure Unified Push.  This involves adding the Unified Push libraries to your Android project, and adding a configuration file to your assets folder.  With the configuration in place, you can begin using the Unified Push SDK. Configuration and usage are discussed beginning with the section *Configure the Unified Push SDK*
+
+### Firebase Configuration with Android Studio
+Android Studio can automate much of the Firebase integration process using the Firebase Assistant.
+
+Access the assistant from the **Tools > Firebase** menu item.
+
+> ![Load Firebase Assistant](assets/android/android_studio_tools_menu_firebase.png) 
```

Comment:
I believe this image is too big. I would suggest to use smaller fonts when getting the screenshot and clip the image at the `Tasks & Contexts` menu item